onnx/onnx-ml.proto

```diff
@@ -242,20 +242,23 @@ message TrainingInfoProto {
   // iteration to zero.
   //
   // By default, this field is an empty graph and its evaluation does not
-  // produce any output.
+  // produce any output. Thus, no initializer would be changed by default.
   optional GraphProto initialization = 1;
 
   // This field represents a training algorithm step. Given required inputs,
   // it computes outputs to update initializers in its own or inference graph's
-  // initializer lists. In general, this graph contains loss node, gradient node,
-  // optimizer node, increment of iteration count, and some calls to the inference
-  // graph.
+  // initializer lists. In general, this field contains loss node, gradient node,
+  // optimizer node, increment of iteration count.
   //
-  // The field algorithm.node is the only place the user can use GraphCall
-  // operator. The only callable graph is the one stored in ModelProto.graph.
+  // In runtime, the training graph, "algorithm", is always combined with
+  // the inference graph, "ModelProto.graph". That is, the actual
+  // input/initializer/output/node/value_info/sparse_initializer list of
+  // training graph is the concatenation of
+  // "ModelProto.graph.input/initializer/output/node/value_info/sparse_initializer"
+  // and "algorithm.input/initializer/output/node/value_info/sparse_initializer".
```

Comment:
Add "in that order. This combined graph must satisfy the normal ONNX conditions. For example, an input of a node in the algorithm graph may reference the output of a node in the inference graph (but not the other way round)."